models/repo_collaboration.go

```diff
@@ -16,16 +16,17 @@ type Collaboration struct {
 	Mode   AccessMode `xorm:"DEFAULT 2 NOT NULL"`
 }
 
-func (c *Collaboration) ModeName() string {
+func (c *Collaboration) ModeI18nKey() string {
 	switch c.Mode {
 	case ACCESS_MODE_READ:
-		return "Read"
+		return "repo.settings.collaboration.read"
 	case ACCESS_MODE_WRITE:
-		return "Write"
+		return "repo.settings.collaboration.write"
 	case ACCESS_MODE_ADMIN:
-		return "Admin"
+		return "repo.settings.collaboration.admin"
+	default:
+		return "repo.settings.collaboration.undefined"
```

Comment:
This was on the bottom of the function before. I find it more Go idiomatic to put this inside the switch block, but the result is the same.